src/transformers/models/swin/modeling_swin.py

```diff
@@ -589,7 +590,6 @@ def __init__(self, config, dim, input_resolution, num_heads, shift_size=0):
         self.shift_size = shift_size
         self.window_size = config.window_size
         self.input_resolution = input_resolution
-        self.set_shift_and_window_size(input_resolution)
```

Comment:
This change is needed to make Swin work with frameworks like OneFormer, UperNet, and it is also backwards compatible with our checkpoints. The shift and window size only needs to be set in the forward pass, not in the init.